react/Button/Readme.md

```diff
@@ -13,12 +13,17 @@ when `<ButtonLink>` has:
 
 ```
 const { Button } = require('./index');
+const props = [{}, { disabled: true}, { busy: true }];
+const themes = ['regular', 'danger', 'highlight', 'secondary', 'danger-outline', 'alpha'];
 
 <div>
-  <p><Button label='normal action' /></p>
-  <p><Button theme='danger' label='erase or destroy' /></p>
-  <p><Button theme='highlight' label='events communication' /></p>
-  <p><Button theme='secondary' label='cancel or second option' /></p>
+  {themes.map(theme => 
+    <p>{
+      props.map(
+        props => <Button label={theme} theme={theme} {...props} />
+      )
+    }</p>
+  )}
```

Comment:
Here I display all themes with disabled and busy to easily debug variants.

Reply:
It appears that `secondary` & `danger-outline` when busy are broken then.

Reply:
Yes they were broken before. I fixed that in https://github.com/cozy/cozy-ui/pull/444/commits/7009bd436f6a7087655650e6e8886dea62ba9187